src/components/PokemonModalWindow/PokemonModalWindow.style.js

```diff
@@ -0,0 +1,29 @@
+import {makeStyles} from "@material-ui/core";
+
+export const styles = (theme) => ({
+    root: {
+        margin: 0,
+        padding: theme.spacing(2),
+        textAlign: 'center'
+    },
+    closeButton: {
+        position: 'absolute',
+        right: theme.spacing(1),
+        top: theme.spacing(1),
+        color: theme.palette.grey[500],
+    },
+});
+
+export const useDialogStyle = makeStyles(() => ({
+    dialogContent: {
+        display: 'flex',
+        flexDirection: "row"
```

Comment:
I think this property is extra. When you set display to flex, flex-direction is row by default.